### PR TITLE
Upgrade matrix-widget-sdk

### PR DESCRIPTION
**Pull request by robintown**
To be merged in lockstep with https://github.com/matrix-org/matrix-react-sdk/pull/8885

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -8222,6 +8222,14 @@ matrix-widget-api@^0.1.0-beta.18:
     "@types/events" "^3.0.0"
     events "^3.2.0"
 
+matrix-widget-api@^1.0.0:
```
Comment (turt2live) [inline yarn.lock:8225]: The lockfile also needs to remove the beta.18 version, otherwise we can end up with two widget-apis in production.

**Review by turt2live — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (turt2live) [reply yarn.lock:8225]: (to do this, update the commit for the resolved version of the react-sdk to your merged react-sdk PR, then `yarn install`. This should cause it to pull the latest definitions and update automatically)

Comment (robintown) [reply yarn.lock:8225]: Thanks, I had to fiddle with both package.json and the lock file and `yarn install --force` for a bit, but it seems to have worked.

**Review by turt2live — APPROVED** [review]
manually also probably works

**Pull request merged**